Builds/containers/centos-builder/Dockerfile

```diff
@@ -1,43 +1,26 @@
-FROM centos:7
+FROM rippleci/centos:7
 ARG GIT_COMMIT=unknown
 ARG CI_USE=false
 
 LABEL git-commit=$GIT_COMMIT
 
 COPY centos-builder/centos_setup.sh /tmp/
-COPY shared/build_deps.sh /tmp/
 COPY shared/install_cmake.sh /tmp/
-COPY centos-builder/extras.sh /tmp/
-COPY shared/install_boost.sh /tmp/
 RUN chmod +x /tmp/centos_setup.sh && \
-    chmod +x /tmp/build_deps.sh && \
-    chmod +x /tmp/install_boost.sh && \
-    chmod +x /tmp/install_cmake.sh && \
-    chmod +x /tmp/extras.sh
+    chmod +x /tmp/install_cmake.sh
 RUN /tmp/centos_setup.sh
 
-RUN /tmp/install_cmake.sh 3.16.1 /opt/local/cmake-3.16
+RUN /tmp/install_cmake.sh 3.16.3 /opt/local/cmake-3.16
 RUN ln -s /opt/local/cmake-3.16 /opt/local/cmake
 ENV PATH="/opt/local/cmake/bin:$PATH"
-# also install min supported cmake for testing
-RUN if [ "${CI_USE}" = true ] ; then /tmp/install_cmake.sh 3.9.0 /opt/local/cmake-3.9; fi
+# TODO: Install latest CMake for testing
+RUN if [ "${CI_USE}" = true ] ; then /tmp/install_cmake.sh 3.16.3 /opt/local/cmake-3.16; fi
```

Comment:
See my comments on the Ubuntu Dockerfile for my thoughts on moving all these `RUN` commands to the install script and greatly simplifying / trivializing the Dockerfile.